rpa-manager/modules/ROOT/pages/processmonitoring-queue.adoc

```diff
@@ -1,6 +1,6 @@
 = Queuing Process Sessions
 
-When more process configurations are deployed than sessions are available, RPA Manager must decide in which order the processes will be executed.
+When more process configurations are deployed than sessions are available, RPA Manager must decide in which order the processes are executed.
```

Comment:
```suggestion
When more process configurations are deployed than sessions are available, RPA Manager must decide in the order in which the processes are executed.
```

Reply:
What you have in the above text isn't wrong, but it sounds awkward.

Reply:
decide the order in which